catalog/models.py

```diff
@@ -1,3 +1,124 @@
+""" catalog models for data storage
+
+The models are defined as classes for every real world object needed in
+the system. Each model class needs to inherid from django.db.models.Model.
+The methods for CRUD (create, read, update and delete) are inherited from
+the super class.
+"""
+
+
 from django.db import models
+import uuid
+from django.db.models.deletion import SET_NULL
+from django.urls import reverse     # Used to generate URLs by reversing the URL patterns
+
+class Genre(models.Model):
+    """Model representing a book genre."""
+
+    name = models.CharField(max_length=200, help_text='Enter a book genre (e.g. Science Fiction)')
```

Comment:
add **blank=False** property will prevent an empty string to be provided in your model.